### ISSUE TITLE
REGISTRY_PROXY_TTL Env Var is not recogized

### ISSUE THREAD

**Issue by AdrianDeWinter** [distribution/distribution#4378]
### Description

When setting proxy.ttl via the env var (REGISTRY_PROXY_TTL), this error is displayed:
`WARN[0000] Ignoring unrecognized environment variable REGISTRY_PROXY_TTL`

### Reproduce

`docker run --rm -it -e REGISTRY_PROXY_REMOTEURL="https://registry-1.docker.io" -e REGISTRY_PROXY_TTL="2400h" registry:2`

### Expected behavior

REGISTRY_PROXY_TTL is the correct format according to the documentation and should set or override the proxy.ttl value

### registry version

Tag: 2.8.3
I bisected the issue and it first crops up in 2.2.0, 2.1.1 is fine

### Additional Info

Other settings under proxy.* work just fine.
Mounting a config file that has proxy.ttl set also avoids the error, and allows proxy.ttl to be overridden via the env var.

**Comment by milosgajdos** [comment]
Please do not use `v2.x` build; it's broken and it no longer receives any updates.

Trys `v3` alpha release from `distribution/distribution` instead of the official image and report back.

**Comment by milosgajdos** [comment]
Closing as dupe of https://github.com/distribution/distribution/issues/4352

**Comment by AdrianDeWinter** [comment]
> Trys `v3` alpha release from `distribution/distribution` instead of the official image and report back.

Thanks for your reply. I tested the alpha image, and the issue does not seem do exist there. Thank you!

> Please do not use `v2.x` build; it's broken and it no longer receives any updates.

A point of (well intentioned) feedback:
It might be good to mention that fact on the docker hub page, which still explicitly recommends the registry:2 image (under "Run a local registry: Quick Version") and the repo's README. I took a quick look a all the closed issues, and the whole "Don't use anything except the alpha" thing comes up all the time. Saves time for all involved :)
